requirements.txt

```diff
@@ -15,6 +15,7 @@ mock==2.0.0
 mysqlclient==1.3.12
 nameparser==0.5.6
 nose==1.3.7
+pandas==0.25.1
```

Comment:
0.25.1 is only supported by Python 3.5+ but the server currently supports 2.7, 3.4, and 3.6. Will talk to @ezwang to see if we still need 2.7 and 3.4.